ui/app/AppLayouts/Chat/components/GroupInfoPopup.qml

```diff
@@ -0,0 +1,152 @@
+            opacity: 0.7
+            font.weight: Font.Bold
+            font.pixelSize: 21
+            color: Theme.white
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+          }
+      }
+    
+      TextEdit {
+          id: groupName
+          text: chatsModel.activeChannel.name
+          anchors.top: parent.top
+          anchors.topMargin: 18
+          anchors.left: letterIdenticon.right
+          anchors.leftMargin: Theme.smallPadding
+          font.bold: true
+          font.pixelSize: 14
+          readOnly: true
+          wrapMode: Text.WordWrap
+      }
+
+      Text {
+          text: {
+            let cnt = chatsModel.activeChannel.members.len();
+            if(cnt > 1) return qsTr("%1 members").arg(cnt);
+            return qsTr("1 member");
+          }
+          width: 160
+          anchors.left: letterIdenticon.right
+          anchors.leftMargin: Theme.smallPadding
+          anchors.top: groupName.bottom
+          anchors.topMargin: 2
+          font.pixelSize: 14
+          color: Theme.darkGrey
+          font.family: "Inter"
+      }
+    }
+
+
+    Item {
+        id: container
+        anchors.fill: parent
+
+        Text {
+            id: memberLabel
+            text: qsTr("Members")
+            anchors.left: parent.left
+            anchors.leftMargin: Theme.padding
+            font.pixelSize: 15
+            color: Theme.darkGrey
+        }
+
+        ListModel {
+            id: exampleModel
+
+            ListElement {
+                isAdmin: false
+                joined: true
+                identicon: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII="
+                userName: "The UserName"
+                pubKey: "0x12345678"
+            }
+
+            ListElement {
+                isAdmin: false
+                joined: true
+                identicon: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII="
+                userName: "The UserName"
+                pubKey: "0x12345678"
+            }
+
+            ListElement {
+                isAdmin: false
+                joined: true
+                identicon: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII="
+                userName: "The UserName"
+                pubKey: "0x12345678"
+            }
+        }
+
+        ListView {
+            id: memberList
+            anchors.fill: parent
+            anchors.top: memberLabel.bottom
+            anchors.bottom: popup.bottom
+            anchors.topMargin: 30
+            anchors.bottomMargin: Theme.padding
+            spacing: 4
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            //model: exampleModel
```

Comment:
Leftover?

Reply:
I left that in case I want to see the example items in the designer